src/Controller/Rest/ApiController.php

```diff
@@ -44,39 +46,76 @@ public function getFreeTimesByDate(string $date)
     {
 
         //Reikia gauti laisvas vietas pagal data ir transforminti i apacioje esanti masyva
-        $dateArr = [
-            '0100-0140' =>  4,
-            '0500-0600' =>  2,
-            '0600-0650' =>  1,
-        ];
+//        $dateArr = [
+//            '0100-0140' =>  1,
```

Comment:
tokiu komentaru kode nerasom. Kodas turi buti kaip gerai parasyta proza - skaitai koda ir tai turi buti suprantama is to kaip jis parasytas, kiekviena karta kai jums reikia komentaro paaiskinti ka daro metodas - jus paskelbiat pasauliui, kad nesugalvojot kaip metoda tinkamai pavadinti